<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Backup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BackupButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Backup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecoverButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Recover</string>
    </property>
   </widget>
   <widget class="QLabel" name="FromLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ToLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SourcePath">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/jefffu/data</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetPath">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/jefffu/data_backup</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseSource">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseTarget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
